utils/utils-lib/src/constants.rs

```diff
@@ -129,3 +129,5 @@ pub const SNAPSHOT_TRANSACTION_PRUNE_LIMIT: usize = 10;
 
 /// Maximum number of snapshot transactions allowed.
 pub const SNAPSHOT_MAX_TRANSACTION_LIMIT: usize = 5;
+/// The default mbus server url.
+pub const DEFAULT_MBUS_SERVER_URL: &str = "nats://mayastor-nats:4222";
```

Comment:
This repo is kubernetes agnostic, shouldn't have a default here, or at least should be based on deployer cluster container name, eg: nats://nats:4222